buildold

```diff
@@ -761,8 +761,8 @@ EOF
     fi
     $DESTINATION/tmp/system_ln $DESTINATION/$newdir $newdir
   done
-  rm -f VERSION
-  $DESTINATION/tmp/system_ln $DESTINATION/include/VERSION VERSION
+  rm -f charm-version.h
+  $DESTINATION/tmp/system_ln $DESTINATION/include/charm-version.h charm-version.h
```

Comment:
Do we also want to remove `charm-version-git.h` here?

Reply:
Not sure, but I think the `rm` refers to the symlink only, and there is no symlink created for `charm-version-git.h`.